www/artworks/get.php

```diff
@@ -11,7 +11,7 @@
 	Template::Emit404();
 }
 
-?><?= Template::Header(['title' => $artwork->Name]) ?>
+?><?= Template::Header(['title' => Formatter::ToPlainText($artwork->Name)]) ?>
```

Comment:
Notice that Template::Header *already* escapes the name. No need to escape it twice.

Reply:
Noted, thanks. Reverted this change.